oracles/oracle_ws_api.md

```diff
@@ -0,0 +1,427 @@
+[back](./oracles.md)
```

Comment:
May I suggest splitting this document in two parts:
* A document that acts as a surrogate for the Swagger schema for websocket
* A document that documents to intended usage of API (the portion "For simplicity, we only work with a single node...")